bin/post_compile

```diff
@@ -5,23 +5,16 @@
 set -euo pipefail
 
 # Make the current Git revision accessible at <site-root>/revision.txt
-echo "$SOURCE_VERSION" > ui/revision.txt
-
-# Create a `dist/` directory containing built/minified versions of the `ui/` assets.
-# Uses the node binaries/packages installed by the nodejs buildpack previously.
-yarn build
+echo "$SOURCE_VERSION" > build/revision.txt
```

Comment:
Since `yarn build` will now already have been run by this point, we must update the revision in the build directory and not in the source directory.